app/models/ems_refresh/save_inventory_cloud.rb

```diff
@@ -171,6 +171,7 @@ def save_key_pairs_inventory(ems, hashes, target = nil)
   def save_cloud_volumes_inventory(ems, hashes, target = nil)
     target = ems if target.nil?
 
+    return unless ems.cloud_volumes
```

Comment:
@hsong-rh If we're delegating `cloud_volumes` for the openstack ems to the Cinder manager, won't this always be true?